components/financial_assistance/app/domain/financial_assistance/operations/applications/pvc/create_pvc_request.rb

```diff
@@ -54,39 +53,100 @@ def create_non_esi_evidences(application)
 
               applicant.create_evidence(:non_esi_mec, "Non ESI MEC")
             end
+            create_evidence_history(application, 'PVC_Submitted', 'PVC - Renewal verifications submitted', 'system')
 
             Success(true)
+          rescue StandardError => e
+            pvc_logger.error("Failed to create non_esi_evidences for application with hbx_id #{application.hbx_id} due to #{e.inspect}")
+            Failure("Failed to create non_esi_evidences for application with hbx_id #{application.hbx_id}")
           end
 
-          def transform_application(application)
-            payload = ::FinancialAssistance::Operations::Applications::Transformers::ApplicationTo::Cv3Application.new.call(application)
-            AcaEntities::MagiMedicaid::Operations::InitializeApplication.new.call(payload.value!)
+          def transform_and_validate_application(application)
+            payload_entity = ::Operations::Fdsh::BuildAndValidateApplicationPayload.new.call(application)
+
+            if EnrollRegistry.feature_enabled?(:validate_and_record_publish_application_errors) && payload_entity.failure?
+              record_application_failure(application, payload_entity.failure.messages)
+              return log_error_and_return_failure(payload_entity.failure.messages)
+            end
+
+            if EnrollRegistry.feature_enabled?(:validate_and_record_publish_application_errors) && payload_entity.success?
+              valid_applicants = validate_applicants(payload_entity, application)
+              return valid_applicants.any?(true) ? payload_entity : log_error_and_return_failure("Failed to transform application with hbx_id #{application.hbx_id} due to all applicants being invalid")
+            end
+
+            payload_entity
           rescue StandardError => e
             pvc_logger.error("Failed to transform application with hbx_id #{application.hbx_id} due to #{e.inspect}")
+            record_application_failure(application, "transformation failure") if payload_entity.failure?
```

Comment:
Instead of checking payload_entity.failure?, we can check  if the feature is enabled